### DISCUSSION TITLE
Support Smart window handle Siegenia ZHSM0020

### DISCUSSION THREAD

**Discussion by hwijers** [Koenkk/zigbee2mqtt#22681] [Devices]
Does anyone here have experience with a Smart window handle ZHSM0020 from Siegenia?

[](https://shop.siegenia.com/siegenia/en/p/ZHSM0020-0M4010)

This is MATTER & THREAD compatible, but would it work with only zigbee? I am now using Zigbee2MQTT with a "Texas Instruments LAUNCHXL-CC26X2R1" Coordinator in a Linux Container and Home Assistant in another Linux Container under Proxmox.

MATTER & THREAD is still a difficult story for me.

**Comment by Koenkk** [comment]
Matter and Thread is not Zigbee, so no, (compare it to Bluetooth and Zigbee, it's just something different).
